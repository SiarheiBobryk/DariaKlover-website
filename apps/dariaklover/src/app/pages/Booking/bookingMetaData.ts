import { PageMetaData } from '@dariaklover/types';

const bookingMetaData: PageMetaData = {
  title: 'Записаться',
  heading: 'Записаться',
  path: '/booking',
};

export default bookingMetaData;
