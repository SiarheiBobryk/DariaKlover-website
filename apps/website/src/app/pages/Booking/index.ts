export { default } from './Booking';

export { default as bookingMetaData } from './bookingMetaData';
