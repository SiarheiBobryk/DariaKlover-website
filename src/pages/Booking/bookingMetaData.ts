export interface BookingMetaData {
  title: string;
  heading: string;
  path: string;
}

const bookingMetaData: BookingMetaData = {
  title: 'Запись на консультацию',
  heading: 'Запись на консультацию (Бета)',
  path: '/booking',
};

export default bookingMetaData;
