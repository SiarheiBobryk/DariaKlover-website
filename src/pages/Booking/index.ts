export { default } from './Booking';

export { default as bookingMetaData } from './bookingMetaData';
export * from './bookingMetaData';
