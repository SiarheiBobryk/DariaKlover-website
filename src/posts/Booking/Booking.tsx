import * as React from 'react';

import Link from '@mui/material/Link';

import OpenInNewIcon from '@mui/icons-material/OpenInNew';
import Collapse from '@mui/material/Collapse';
import Alert from '@mui/material/Alert';
import AlertTitle from '@mui/material/AlertTitle';
import IconButton from '@mui/material/IconButton';
import CloseIcon from '@mui/icons-material/Close';

import { AppConfigContext, Config } from '../../providers/AppConfigProvider';
import Heading from '../../components/Heading';
import Paragraph from '../../components/Paragraph';

export interface BookingProps {
  heading?: string;
}

function Booking(props: BookingProps) {
  const { heading } = props;
  const {
    calendly: { label: calendlyLabel, href: calendlyHref },
  }: Config = React.useContext(AppConfigContext);
  const [openInfo, setOpenInfo] = React.useState(true);

  return (
    <>
      <Heading>{heading}</Heading>
      <Paragraph>
        –ò—Å–ø–æ–ª—å–∑—É–π—Ç–µ{' '}
        <Link sx={{ textTransform: 'lowercase' }} href={calendlyHref} target="_blank">
          {calendlyLabel} <OpenInNewIcon fontSize="small" sx={{ verticalAlign: 'middle' }} />
        </Link>{' '}
        –¥–ª—è –æ–∑–Ω–∞–∫–æ–º–ª–µ–Ω–∏—è —Å —Ä–∞—Å–ø–∏—Å–∞–Ω–∏–µ–º –∏ –¥–ª—è –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é.
      </Paragraph>
      <Collapse in={openInfo}>
        <Alert
          severity="info"
          action={
            <IconButton
              aria-label="–ó–∞–∫—Ä—ã—Ç—å"
              color="inherit"
              onClick={() => {
                setOpenInfo(false);
              }}
            >
              <CloseIcon fontSize="inherit" />
            </IconButton>
          }
          sx={{ mb: 2 }}
        >
          <AlertTitle>–ú–∞–ª–µ–Ω—å–∫–æ–µ –æ–±—ä—è–≤–ª–µ–Ω–∏–µ</AlertTitle>
          –ö–∞–ª–µ–Ω–¥–∞—Ä—å –î–∞—Ä—å–∏ –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –Ω–∞ —Å—Ç–æ—Ä–æ–Ω–Ω–µ–º —Å–µ—Ä–≤–∏—Å–µ Calendly, –∫–æ—Ç–æ—Ä—ã–π –æ—á–µ–Ω—å –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω—ã–π. –í —Ç–æ –∂–µ –≤—Ä–µ–º—è, –∫
          —Å–æ–∂–∞–ª–µ–Ω–∏—é, —ç—Ç–æ—Ç —Å–µ—Ä–≤–∏—Å –Ω–µ –∏–º–µ–µ—Ç —Ä—É—Å—Å–∫–æ–≥–æ —è–∑—ã–∫–∞. –î–ª—è —Ç–µ—Ö, –∫—Ç–æ –Ω–µ –≥–æ–≤–æ—Ä–∏—Ç –ø–æ-–∞–Ω–≥–ª–∏–π—Å–∫–∏, –º—ã –æ—á–µ–Ω—å –ø–æ—Å—Ç–∞—Ä–∞–ª–∏—Å—å
          –ø–µ—Ä–µ–≤–µ—Å—Ç–∏ –Ω–∞ —Ä—É—Å—Å–∫–∏–π —è–∑—ã–∫ –≤—Å–µ —Å—Ç—Ä–∞–Ω–∏—á–∫–∏, –Ω–æ, –∫ —Å–æ–∂–∞–ª–µ–Ω–∏—é, –Ω–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç —Ç–µ—Ö–Ω–∏—á–µ—Å–∫–∏ –Ω–µ –≤–æ–∑–º–æ–∂–Ω–æ –ø—Ä–µ–¥–æ—Å—Ç–∞–≤–∏—Ç—å
          100% —Ä—É—Å—Å–∫—É—é –≤–µ—Ä—Å–∏—é, –ø–æ—ç—Ç–æ–º—É –º—ã –∞–∫—Ç–∏–≤–Ω–æ —Ä–∞–±–æ—Ç–∞–µ–º –Ω–∞–¥ —Ä–∞–∑—Ä–∞–±–æ—Ç–∫–æ–π –∏–Ω—Å—Ç—Ä—É–∫—Ü–∏–∏ –ø–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞–Ω–∏—é –∫–∞–ª–µ–Ω–¥–∞—Ä—è –î–∞—Ä—å–∏,
          –∫–æ—Ç–æ—Ä–∞—è –æ—á–µ–Ω—å —Å–∫–æ—Ä–æ –±—É–¥–µ—Ç –æ–ø—É–±–ª–∏–∫–æ–≤–∞–Ω–∞ –Ω–∞ —ç—Ç–æ–π —Å—Ç—Ä–∞–Ω–∏—Ü–µ. –°–ø–∞—Å–∏–±–æ –∑–∞ –≤–∞—à–µ —Ç–µ—Ä–ø–µ–Ω–∏–µ –∏ –ø–æ–Ω–∏–º–∞–Ω–∏–µ üòá
        </Alert>
      </Collapse>
    </>
  );
}

export default Booking;
