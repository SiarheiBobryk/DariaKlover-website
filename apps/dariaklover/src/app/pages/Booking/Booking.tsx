import * as React from 'react';
import { Helmet } from 'react-helmet';
import { NavLink as RouterNavLink } from 'react-router-dom';

import Alert from '@mui/material/Alert';
import AlertTitle from '@mui/material/AlertTitle';
import Grid from '@mui/material/Grid';
import Link from '@mui/material/Link';
import Typography from '@mui/material/Typography';

import { CalendlyContext } from '../../providers';
import { CalendlyUser, CalendlyUserEventType, getCalendlyEventsActive } from '../../services/calendlyUserService';
import BookingCard from '../../components/BookingCard';
import bookingMetaData from './bookingMetaData';
import Heading from '../../components/Heading';
import helpMetaData from '../Help/helpMetaData';
import howToBookMetaData from '../HowToBook/howToBookMetaData';
import LinkEmail from '../../components/LinkEmail';
import LinkSocial from '../../components/LinkSocial';
import Paragraph from '../../components/Paragraph';

function Booking() {
  const calendlyUser: CalendlyUser = React.useContext<CalendlyUser>(CalendlyContext);
  const [events, setEvents] = React.useState<Array<CalendlyUserEventType>>([]);

  const fetchCalendlyEvents = React.useCallback(
    async function asyncFetchCalendlyEvents() {
      const calendlyEvents: Array<CalendlyUserEventType> = await getCalendlyEventsActive(calendlyUser.uri);
      setEvents(calendlyEvents);
    },
    [calendlyUser?.uri],
  );

  React.useEffect(
    function updateCalendlyEvents() {
      fetchCalendlyEvents().catch(function catchError(error) {
        // eslint-disable-next-line no-console
        console.error(error);
      });
    },
    [fetchCalendlyEvents],
  );

  return (
    <>
      <Helmet>
        <title>{bookingMetaData.title}</title>
      </Helmet>
      <Heading>{bookingMetaData.heading}</Heading>
      {events.length > 0 ? (
        <Grid container spacing={1} sx={{ marginBottom: 1 }}>
          {events.map(function generateEvents(event) {
            return (
              <Grid key={event.uri} item xs={12} sm={6}>
                <BookingCard title={event.name} duration={event.duration} href={event.scheduling_url} uri={event.uri}>
                  <Typography
                    variant="body2"
                    color="text.secondary"
                    dangerouslySetInnerHTML={{ __html: event.description_html }}
                  />
                </BookingCard>
              </Grid>
            );
          })}
        </Grid>
      ) : (
        <Paragraph>
          К сожалению, на данный момент нет доступных консультаций. Скорее всего это происходит по техническим причинам.
          Например, в этот момент выполняется перенастройка календаря или идет техническое обслуживание. Попробуйте
          открыть эту страницу через несколько минут. Если после этого запись не возобновилась, пожалуйста, сообщите мне
          через <LinkSocial social="instagram" />, <LinkSocial social="telegram" />, <LinkSocial social="whatsapp" />{' '}
          или по <LinkEmail />. Мы обязательно что-нибудь придумаем.
        </Paragraph>
      )}
      <Alert severity="warning" variant="outlined">
        <AlertTitle>Пропуск запланированной встречи</AlertTitle>
        Если вы не можете прийти на запланированную встречу, отмените или перенесите ее на другой день не менее чем за
        24 ч. (см раздел{' '}
        <Link component={RouterNavLink} to={howToBookMetaData.path}>
          {howToBookMetaData.heading}
        </Link>{' '}
        и{' '}
        <Link component={RouterNavLink} to={helpMetaData.path}>
          {helpMetaData.heading}
        </Link>
        ). Если вы записались, но не явились на встречу, вы автоматически теряете возможность записи к Дарье в
        последующем.
      </Alert>
      <Alert severity="info" variant="outlined">
        <AlertTitle>{howToBookMetaData.heading}</AlertTitle>
        Если вам нужна помощь в том, как записаться на консультацию, пожалуйста, воспользуйтесь инструкцией{' '}
        <Link component={RouterNavLink} to={howToBookMetaData.path}>
          {howToBookMetaData.heading}
        </Link>
        .
      </Alert>
    </>
  );
}

export default Booking;
