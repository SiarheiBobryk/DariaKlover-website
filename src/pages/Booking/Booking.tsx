import * as React from 'react';
import { Helmet } from 'react-helmet';
import { useSearchParams } from 'react-router-dom';

import Alert from '@mui/material/Alert';
import AlertTitle from '@mui/material/AlertTitle';

import List from '@mui/material/List';
import ListItem from '@mui/material/ListItem';
import Link from '@mui/material/Link';
import Heading from '../../components/Heading';
import Paragraph from '../../components/Paragraph';
import { AppConfigContext, Config } from '../../providers/AppConfigProvider';
import bookingMetaData from './bookingMetaData';

const calendlyResponceKeys = [
  'assigned_to',
  'event_type_uuid',
  'event_type_name',
  'event_start_time',
  'event_end_time',
  'invitee_uuid',
  'invitee_full_name',
  'invitee_email',
  'text_reminder_number',
];

function References() {
  const {
    calendly: { href: calendlyHref },
  }: Config = React.useContext(AppConfigContext);

  const [searchParams, setSearchParams] = useSearchParams();

  console.info(searchParams.get('assigned_to'));

  return (
    <>
      <Helmet>
        <title>{bookingMetaData.title}</title>
      </Helmet>
      <Heading>{bookingMetaData.heading}</Heading>
      <Paragraph>–ù–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç –¥–æ—Å—Ç—É–ø–Ω—ã —Å–ª–µ–¥—É—é—â–∏–µ –≤–∏–¥—ã –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–π:</Paragraph>
      <List>
        <ListItem>
          <Link href={`${calendlyHref}/trial`}>–û–∑–Ω–∞–∫–æ–º–∏—Ç–µ–ª—å–Ω–∞—è –≤—Å—Ç—Ä–µ—á–∞</Link>
        </ListItem>
        <ListItem>
          <Link href={`${calendlyHref}/regular`}>Heath-–∫–æ—É—á–∏–Ω–≥ —Å–µ—Å—Å–∏—è</Link>
        </ListItem>
      </List>
      <Alert sx={{ my: 1 }} severity="warning">
        <AlertTitle>–°–∏—Å—Ç–µ–º–∞ —Ç–µ—Å—Ç–∏—Ä—É–µ—Ç—Å—è</AlertTitle>
        –û–±—Ä–∞—â–∞–µ–º –≤–∞—à–µ –≤–Ω–∏–º–∞–Ω–∏–µ, —á—Ç–æ —Å–∏—Å—Ç–µ–º–∞ –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –≤ —Ä–µ–∂–∏–º–µ —Ç–µ—Å—Ç–∏—Ä–æ–≤–∞–Ω–∏—è –∏ –º–æ–∂–µ—Ç –±—ã—Ç—å
        –Ω–µ—Å—Ç–∞–±–∏–ª—å–Ω–∞.
      </Alert>
      <Alert sx={{ my: 1 }} severity="info">
        <AlertTitle>–†—É—Å—Å–∫–∏–π —è–∑—ã–∫</AlertTitle>
        C–∏—Å—Ç–µ–º–∞ –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –Ω–∞ —Å—Ç–æ—Ä–æ–Ω–Ω–µ–º —Å–µ—Ä–≤–∏—Å–µ, –∫–æ—Ç–æ—Ä—ã–π –æ—á–µ–Ω—å –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω—ã–π, –æ–¥–Ω–∞–∫–æ —ç—Ç–æ—Ç
        —Å–µ—Ä–≤–∏—Å –Ω–µ –∏–º–µ–µ—Ç —Ä—É—Å—Å–∫–æ–≥–æ —è–∑—ã–∫–∞. –î–ª—è —Ç–µ—Ö, –∫—Ç–æ –Ω–µ –≥–æ–≤–æ—Ä–∏—Ç –ø–æ-–∞–Ω–≥–ª–∏–π—Å–∫–∏, –º—ã –æ—á–µ–Ω—å –ø–æ—Å—Ç–∞—Ä–∞–ª–∏—Å—å –ø–µ—Ä–µ–≤–µ—Å—Ç–∏ –Ω–∞ —Ä—É—Å—Å–∫–∏–π
        —è–∑—ã–∫ –≤—Å–µ —ç–ª–µ–º–µ–Ω—Ç—ã –Ω–∞ —Å—Ç—Ä–∞–Ω–∏—á–∫–∞—Ö, –Ω–æ, –∫ —Å–æ–∂–∞–ª–µ–Ω–∏—é, –Ω–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç —Ç–µ—Ö–Ω–∏—á–µ—Å–∫–∏ –Ω–µ –≤–æ–∑–º–æ–∂–Ω–æ –ø—Ä–µ–¥–æ—Å—Ç–∞–≤–∏—Ç—å 100%
        —Ä—É—Å—Å–∫—É—é –≤–µ—Ä—Å–∏—é, –ø–æ—ç—Ç–æ–º—É –º—ã –∞–∫—Ç–∏–≤–Ω–æ —Ä–∞–±–æ—Ç–∞–µ–º –Ω–∞–¥ —Ä–∞–∑—Ä–∞–±–æ—Ç–∫–æ–π –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω–æ–π –∏–Ω—Å—Ç—Ä—É–∫—Ü–∏–∏ –ø–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞–Ω–∏—é,
        –∫–æ—Ç–æ—Ä–∞—è —Å–∫–æ—Ä–æ –±—É–¥–µ—Ç –æ–ø—É–±–ª–∏–∫–æ–≤–∞–Ω–∞ –Ω–∞ —ç—Ç–æ–π —Å—Ç—Ä–∞–Ω–∏—Ü–µ. –°–ø–∞—Å–∏–±–æ –∑–∞ –≤–∞—à–µ —Ç–µ—Ä–ø–µ–Ω–∏–µ –∏ –ø–æ–Ω–∏–º–∞–Ω–∏–µ üòá
      </Alert>
    </>
  );
}

export default References;
