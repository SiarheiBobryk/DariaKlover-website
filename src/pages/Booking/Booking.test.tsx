import * as React from 'react';
import { Helmet, HelmetPropsToState } from 'react-helmet';

import { render, RenderResult } from '@testing-library/react';
import renderer, { ReactTestRendererJSON } from 'react-test-renderer';
import { axe, toHaveNoViolations } from 'jest-axe';

import Booking from './Booking';
import bookingMetaData from './bookingMetaData';

expect.extend(toHaveNoViolations);

describe('<Booking />', () => {
  describe('a11y', () => {
    it('should match a11y standards', async () => {
      const { container }: RenderResult = render(<Booking />);

      const results = await axe(container);
      expect(results).toHaveNoViolations();
    });
  });

  describe('snapshotting', () => {
    it('should render correctly', () => {
      const about: ReactTestRendererJSON | ReactTestRendererJSON[] | null = renderer.create(<Booking />).toJSON();
      expect(about).toMatchSnapshot();
    });
  });

  describe('Elements inspection', () => {
    it('should have the right title HTML element', () => {
      render(<Booking />);

      const { title }: HelmetPropsToState = Helmet.peek();
      expect(title).toBe(bookingMetaData.title);
    });

    it('should have the `h2` HTML element as heading', () => {
      render(<Booking />);

      const h2: HTMLHeadingElement | null = document.querySelector('h2');
      expect(h2).toBeInTheDocument();
      expect(h2).toHaveTextContent(bookingMetaData.heading);
    });
  });
});
