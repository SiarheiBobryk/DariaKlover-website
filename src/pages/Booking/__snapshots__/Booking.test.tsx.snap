// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Booking /> snapshotting should render correctly 1`] = `
Array [
  <h2
    className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
  >
    –ó–∞–ø–∏—Å—å –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é (–ë–µ—Ç–∞)
  </h2>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –ù–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç –¥–æ—Å—Ç—É–ø–Ω—ã —Å–ª–µ–¥—É—é—â–∏–µ –≤–∏–¥—ã –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–π: 
  </p>,
  <ul
    className="MuiList-root MuiList-padding css-h4y409-MuiList-root"
  >
    <li
      className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
      disabled={false}
    >
      <a
        className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
        href="https://calendly.com/dariaklover/trial"
        onBlur={[Function]}
        onFocus={[Function]}
        target="_blank"
      >
        –û–∑–Ω–∞–∫–æ–º–∏—Ç–µ–ª—å–Ω–∞—è –≤—Å—Ç—Ä–µ—á–∞
      </a>
    </li>
  </ul>,
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardWarning MuiAlert-standard css-5ubuof-MuiPaper-root-MuiAlert-root"
    role="alert"
  >
    <div
      className="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
        data-testid="ReportProblemOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 5.99L19.53 19H4.47L12 5.99M12 2L1 21h22L12 2zm1 14h-2v2h2v-2zm0-6h-2v4h2v-4z"
        />
      </svg>
    </div>
    <div
      className="MuiAlert-message css-acap47-MuiAlert-message"
    >
      <div
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
      >
        –°–∏—Å—Ç–µ–º–∞ —Ç–µ—Å—Ç–∏—Ä—É–µ—Ç—Å—è
      </div>
      –û–±—Ä–∞—â–∞–µ–º –≤–∞—à–µ –≤–Ω–∏–º–∞–Ω–∏–µ, —á—Ç–æ —Å–∏—Å—Ç–µ–º–∞ –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –≤ —Ä–µ–∂–∏–º–µ —Ç–µ—Å—Ç–∏—Ä–æ–≤–∞–Ω–∏—è –∏ –º–æ–∂–µ—Ç –±—ã—Ç—å –Ω–µ—Å—Ç–∞–±–∏–ª—å–Ω–∞.
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardInfo MuiAlert-standard css-efxmhs-MuiPaper-root-MuiAlert-root"
    role="alert"
  >
    <div
      className="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
        data-testid="InfoOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M11,9H13V7H11M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20, 12C20,16.41 16.41,20 12,20M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10, 10 0 0,0 12,2M11,17H13V11H11V17Z"
        />
      </svg>
    </div>
    <div
      className="MuiAlert-message css-acap47-MuiAlert-message"
    >
      <div
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
      >
        –†—É—Å—Å–∫–∏–π —è–∑—ã–∫
      </div>
      C–∏—Å—Ç–µ–º–∞ –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –Ω–∞ —Å—Ç–æ—Ä–æ–Ω–Ω–µ–º —Å–µ—Ä–≤–∏—Å–µ, –∫–æ—Ç–æ—Ä—ã–π –æ—á–µ–Ω—å –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω—ã–π, –æ–¥–Ω–∞–∫–æ —ç—Ç–æ—Ç —Å–µ—Ä–≤–∏—Å –Ω–µ –∏–º–µ–µ—Ç —Ä—É—Å—Å–∫–æ–≥–æ —è–∑—ã–∫–∞. –î–ª—è —Ç–µ—Ö, –∫—Ç–æ –Ω–µ –≥–æ–≤–æ—Ä–∏—Ç –ø–æ-–∞–Ω–≥–ª–∏–π—Å–∫–∏, –º—ã –æ—á–µ–Ω—å –ø–æ—Å—Ç–∞—Ä–∞–ª–∏—Å—å –ø–µ—Ä–µ–≤–µ—Å—Ç–∏ –Ω–∞ —Ä—É—Å—Å–∫–∏–π —è–∑—ã–∫ –≤—Å–µ —ç–ª–µ–º–µ–Ω—Ç—ã –Ω–∞ —Å—Ç—Ä–∞–Ω–∏—á–∫–∞—Ö, –Ω–æ, –∫ —Å–æ–∂–∞–ª–µ–Ω–∏—é, –Ω–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç —Ç–µ—Ö–Ω–∏—á–µ—Å–∫–∏ –Ω–µ –≤–æ–∑–º–æ–∂–Ω–æ –ø—Ä–µ–¥–æ—Å—Ç–∞–≤–∏—Ç—å 100% —Ä—É—Å—Å–∫—É—é –≤–µ—Ä—Å–∏—é, –ø–æ—ç—Ç–æ–º—É –º—ã –∞–∫—Ç–∏–≤–Ω–æ —Ä–∞–±–æ—Ç–∞–µ–º –Ω–∞–¥ —Ä–∞–∑—Ä–∞–±–æ—Ç–∫–æ–π –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω–æ–π –∏–Ω—Å—Ç—Ä—É–∫—Ü–∏–∏ –ø–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞–Ω–∏—é, –∫–æ—Ç–æ—Ä–∞—è —Å–∫–æ—Ä–æ –±—É–¥–µ—Ç –æ–ø—É–±–ª–∏–∫–æ–≤–∞–Ω–∞ –Ω–∞ —ç—Ç–æ–π —Å—Ç—Ä–∞–Ω–∏—Ü–µ. –°–ø–∞—Å–∏–±–æ –∑–∞ –≤–∞—à–µ —Ç–µ—Ä–ø–µ–Ω–∏–µ –∏ –ø–æ–Ω–∏–º–∞–Ω–∏–µ üòá
    </div>
  </div>,
]
`;
